src/resolvers.ts

```diff
@@ -23,3 +35,18 @@ export const resolvers = {
     },
   },
 };
+
+function validPassword(password: string): boolean {
+  const regex = /^(?=.*[A-Za-z])(?=.*\d).{6,}$/;
+  return regex.test(password);
+}
+
+async function duplicateEmail(email: string) {
+  const userRepository = AppDataSource.getRepository(User);
+  const user = await userRepository.find({
+    where: {
+      email: email,
+    },
+  });
+  return user.length;
```

Comment:
Acredito que faça sentido essa função retornar um boolean no lugar de um número. Quando alguém for usar essa função, não ficaria claro o que significaria um número sem abrir e ler o que a função faz. Para corrigir isso, só precisaria transformar para boolean
```suggestion
  return !!user.length;
```